tests/unit/components/Scorecard.spec.ts

```diff
@@ -96,6 +97,13 @@ describe("Scorecard.vue", () => {
       isShown: true,
     });
     await flushPromises();
+    const confetti = require("canvas-confetti");
+    const confettiCanvas = document.getElementById("confetticanvas");
+    const confettiHandler = confetti.create(confettiCanvas, {
+      resize: true,
+    });
+    throwConfetti(confettiHandler);
+    expect(throwConfetti).toHaveBeenCalled();
```

Comment:
think logically - this is a test - how is it supposed to happen? How have previous method calls been tested? Feels like I am repeating myself a lot :/